ops/lib.rs

```diff
@@ -541,11 +585,13 @@ fn is_fast_scalar(
     return Some(quote! { #core::v8::fast_api::#cty::Void });
   }
   // TODO(@littledivy): Support u8, i8, u16, i16 by casting.
+
   match tokens(&ty).as_str() {
     "u32" => Some(quote! { #core::v8::fast_api::#cty::Uint32 }),
     "i32" => Some(quote! { #core::v8::fast_api::#cty::Int32 }),
     "f32" => Some(quote! { #core::v8::fast_api::#cty::Float32 }),
     "f64" => Some(quote! { #core::v8::fast_api::#cty::Float64 }),
+    "bool" => Some(quote! { #core::v8::fast_api::#cty::Bool }),
```

Comment:
I've tried reading online and still haven't managed to explain to myself if it's actually _really_ safe to use Rust `bool` as a C++ `bool` but it has seemed to work, so I think it's all fine :D

Reply:
its safe to assume I guess. https://rust-lang.github.io/unsafe-code-guidelines/layout/scalars.html#bool

Reply:
Yeah, that's my understanding as well :)